### DISCUSSION TITLE
Modelleerprinrcipes - U5. Informatiemodel is afgestemd op gebruik door de doelgroepen

### DISCUSSION THREAD

**Discussion by RiX012** [Geonovum/disgeo-imsor#43] [General] [answered]
In dit uitgangspunt wordt gesteld:

>Als wij kunnen definieren voor wie ons informatiemodel is, kunnen we een soortgelijk principe definiëren. We hebben het af en toe >over 'dingen zuiver modelleren'. Dat is zoiets als 'theoretical purity' in het lijstje hierboven. Niet onbelangrijk, maar moet lagere >prioriteit hebben dan andere belangen. Welke belangen zijn dit? Als we dit helder kunnen formuleren, kan ons dit helpen bij het >nemen van beargumenteerde ontwerpbeslissingen.

In principe ben ik het er wel mee eens. Tegelijkertijd wijst de praktijk uit dat de gebruikers vaak (nog) niet de juiste vraag kunnen stellen of het juiste gebruik kunnen voorzien. De afweging tussen 'belang van gebruikers' en 'zuiver modelleren' is daarmee imo niet zo zwart-wit en dient elke keer afgewogen te worden. Zuiver modelleren is voor mij ook: committeren aan een standaard. Dat is op de lange termijn nodig als je de wendbare gebruiker wilt blijven kunnen voorzien.

**Comment by lvdbrink** [comment]
> Tegelijkertijd wijst de praktijk uit dat de gebruikers vaak (nog) niet de juiste vraag kunnen stellen of het juiste gebruik kunnen voorzien.

Dat is waar. Ik probeer altijd de 'vraag achter de vraag' duidelijk te krijgen. Maar dat is niet altijd goed mogelijk. 

Dit principe is bedoeld om te voorkomen dat we vooral iets gaan maken dat we zelf heel mooi vinden. Zoals ik in NEN 2660 las: 'de praktijk boven de theorie'. Dat sluit ook aan bij wat we bedoelen met dit principe. De kunst is om dit goed te verwoorden waarbij we de nuance die jij aanbrengt ook terug laten komen.

**Reply by RiX012** [reply]
Nu ik er nog even over nagedacht heb gaat het me inderdaad niet alleen/zozeer om de 'vraag achter de vraag', maar nog meer over het committeren aan standaarden. Zoals ook in meeting 11 ook al besproken. De balans tussen gebruiken wat er in de standaard staat, t.o.v. hoe gebruikers het zouden willen vind ik vaak verkeerd uitvallen. Gebruiken wat de standaard voorschrijft heeft veel voordelen, zeker als het gaat om het samenbrengen van meerdere registraties. Het is vaak een duurzamere en meer te onderbouwen way-to-go dan 'maatwerk'.

**Comment by pmaria** [comment]
Discussie uit expertgroepsessie:
> - Michel: beter van balans spreken, niet prioriteit.
> - Lennart: deels mee eens, wel een blocking issue met de prioriteit: als je een SOR in nederland wil realiseren heb je meer aandacht nodig voor de semantiek/goed bedenken van de betekenis van dingen - dit zou dan boven de gebruikers moeten staan.
> - Rik: dit is iets wat meer tractie moet krijgen, zodat je maatwerk oplossingen uit de weg gaat. Vaak wijk je af van standaarden wanneer je opdrachtgever iets vraagt, eigenlijk zou je het omkeren en willen zeggen tegen gebruikers: heb je wel deze behoefte? (Dus misschien ook belangrijk om te voorkomen dat je aan allerlei exotische wensen/eisen gaat voldoen.)
> - Jan: [over 'overheidsgebruikers']: is dat inclusief "bedrijven met een publieke taak (zoals bv. Netbeheerders)? Handig deze ook mee te nemen, aangezien ze ook in de categorie 'verplicht gebruik' zitten.

**Comment by lvdbrink** [comment]
In het kader van **balans**: Misschien kunnen we iets toevoegen als **"de belangen van de doelgroepen wegen we af tegen het overkoepelende doel van de SOR, het kunnen gebruiken van de geobasisreg gegevens in samenhang".**

**Reply by pmaria** [reply]
Wellicht is het ook goed om te duiden hoe dit uitgangspunt in de praktijk wordt gebracht. Het zou natuurlijk raar zijn dat  het invullen van een gebruikerswens leidt tot problemen voor alle andere "constituencies".

**Reply by RiX012** [reply]
Persoonlijk zou ik er niet vies van zijn om het helemaal om te draaien. Dus "omdat we met veel stakeholders en verschillende gebruikers te maken hebben, zetten we commitment aan de standaarden voor op. Dit om het duurzaam en uitlegbaar te maken. Wanneer een gebruikerswens niet voldaan kan worden moet afgewogen worden of de gebruikerswens generiek genoeg is op de lange termijn"

**Comment by lvdbrink** [comment]
Na expertgroepbijeenkomst 17/7: Hier is nog geen consensus over. We kunnen wellicht beter over balans spreken, niet prioriteiten. Punt 4 moeten we anders verwoorden, nu lijkt het een kritiek op dit process. 

- Rik: er zit een verschil tussen zuiver modelleren en je committeren aan standaarden voor het modelleren. Modelleren volgens de standaarden is belangrijk en zou geen lage prioriteit moeten hebben. Gebruikers denken vaak aan hun belang in nauwere zin terwijl standaarden uiteindelijk ook in het belang van gebruikers zijn.
- Linda: dat we ons committeren aan standaarden staat al in een ander uitgangspunt. ik bedoelde met dit uitgangspunt niet dat we alles wat gebruikers roepen, voorrang geven bij het modelleren. Wij maken daarin wel een eigen afweging. Maar ik maak geen informatiemodel om een mooi informatiemodel te maken, het is geen doel op zich. Ik doe het uiteindelijk omdat gebruikers het nodig hebben. En wil het daarom bijvoorbeeld niet onnodig complex maken of er inhoud aan toevoegen die gebruikers niet nodig hebben.
- [door wie is dit gezegd?] We zouden kunnen zeggen: voor het modelleren zelf doen we geen concessies. Wanneer er toch iets is wat niet in de standaarden (NEN3610/NEN2660/etc) beschreven wordt, dan gaan we een afweging maken van de belangen van verschillende groepen om tot een beslissing te komen.

**Comment by lvdbrink** [comment, marked as answer]
[U5](https://geonovum.github.io/disgeo-imsor/modelleerprincipes/#u5-informatiemodel-is-afgestemd-op-gebruik-door-de-doelgroepen) is na commentaar van de expertgroep in de laatste iteratie herschreven: 
- het principe dat het ene belang prioriteit heeft boven het andere, is eruit gehaald. Het zijn nu 'belangen' die worden 'afgewogen' met als doel te helpen bij het maken van keuzen tussen verschillende modelleeropties. 
- 'zuiver modelleren' is verwijderd. Dit zit al voldoende in het belang van de informatiemodelleurs.

**Comment by lvdbrink** [comment]
U5 is vastgesteld in [expertgroep 8 juli](https://github.com/Geonovum/disgeo-imsor/blob/master/overleg/08-07-2021-aantekeningen.md).
